thunder-service/job/aws_instances.go

```diff
@@ -32,24 +32,21 @@ func (job *AwsDataRetrieveJob) FetchAwsNotActiveInstances(profile model.AwsProfi
 
 	listRec := make([]interface{}, 0)
 
-	sess, err := session.NewSession(&aws.Config{
-		Region:      aws.String(profile.Region),
-		Credentials: credentials.NewStaticCredentials(profile.AccessKeyId, *profile.SecretAccessKey, ""),
-	})
+	cfg, err := job.loadDefaultConfig(profile)
 	if err != nil {
 		return err
 	}
 
-	ec2Svc := ec2.New(sess)
+	ec2Client := ec2.NewFromConfig(cfg)
 
-	resultec2Svc, err := ec2Svc.DescribeInstances(nil)
+	resultec2Svc, err := ec2Client.DescribeInstances(context.Background(), nil)
 	if err != nil {
 		return err
 	}
 
 	for _, w := range resultec2Svc.Reservations {
 		for _, i := range w.Instances {
-			if *i.State.Name == "stopped" {
+			if i.State.Name == ec2Types.InstanceStateNameStopped {
```

Comment:
using the const `ec2Types.InstanceStateNameStopped` is a better way to check !
just make sure `i.State` is not nil before access to `name`